pytest.ini

```diff
@@ -3,3 +3,5 @@ norecursedirs = .git .* static
 addopts = -rsxX --showlocals --tb=native --nomigrations
 DJANGO_SETTINGS_MODULE = atmo.settings
 DJANGO_CONFIGURATION = Test
+blockade = True
+; blockade-http-whitelist = accounts.google.com
```

Comment:
Yeah, mostly a leftover, but it's also a reminder that this is possible. We need some proper docs eventually